### ISSUE TITLE
Foldseek easy-cluster, easy-search

### ISSUE THREAD

**Issue by jmoojun** [steineggerlab/foldseek#439]
Hi, thank you for making such a nice tool. I'm trying to do 2 things to my test set using foldseek.

The first is clustering my test set to leave only representatives, using structure-based clustering. My test set mmcifs are in the directory `cifs` Initially I used this command:
`foldseek easy-cluster cifs/ foldseek tmp -c 0.8 --min-seq-id 0.3
`
But I thought I didn't add the TM-score metric to cluster my test set, so I added the tmscore-threshold argument to cluster mmcifs with >0.5 TM-score. And I want the TM-score to be calculated by both query and template ((qTM + tTM) / 2). Would changing to this command work in this situation? (I added min-seq-id 0 because I only wanted to use TM-score for clustering)
`foldseek easy-cluster cifs/ foldseek tmp -c 0.8 --min-seq-id 0 --tmscore-threshold 0.5 --tmalign-hit-order 0`
In the original command I sent you, how is foldseek clustering without the tmscore threshold?

The second is searching through the alphafold training set (until 2018-04-30) and filter out representatives with lDDT>0.55 with any of the training set proteins. My representatives will be in the `representatives` directory, and the training set mmcifs are in the `training_set` directory. Would using this command be fit?
`foldseek easy-search representatives/ training_set/ aln tmp -c 0.8 --lddt-threshold 0.55`
